klibs.md

```diff
@@ -13,7 +13,7 @@ The allowed configuration properties are:
 - `ntpPort` - the ntp sever port;
 - `ntpPollMin` - the minimum poll time is expressed as a power of two. The default value is 6, corresponding to 64 seconds (2^6 = 64);
 - `ntpPollMax` - the maximum poll time is expressed as a power of two. The default value is 10, corresponding to 1024 seconds (2^10 = 1024).
-- `ntpResetThreshold` - this is a threshold to use step/jump versus smearing on ntp - the default is set to 0 meaning it will never step/jump
+- `ntpResetThreshold` - this is a threshold expressed in seconds to use step/jump versus smearing on ntp - the default is set to 0 meaning it will never step/jump
```

Comment:
Maybe "difference threshold expressed in seconds"? Perhaps a little harder to read. Not sure if measuring the time difference is already implied.